packages/utils/src/__tests__/joiner/remote-joiner-data.ts

```diff
@@ -0,0 +1,466 @@
+import { MedusaContainer, RemoteJoinerQuery } from "@medusajs/types"
+import { RemoteJoiner } from "../../joiner"
+
+import {
+  mock,
+  serviceConfigs,
+  serviceMock,
+} from "../../__mocks__/joiner/mock_data"
+
+const container = {
```

Comment:
discussion: I would have suggest to create a container using awilix in normal time but I think that it is good to avoid that in the utils package to later get rid of those big deps in the utils package. See global comments